framework/tlvf/tlvf.py

```diff
@@ -1007,10 +1007,6 @@ def addClassVarLenMethods(self, obj_meta, param_type, param_name, param_meta, pa
             lines_cpp.append( "%sreturn false;" % self.getIndentation(2))
             lines_cpp.append( "%s}" % self.getIndentation(1) )
 
-            lines_cpp.append( "%sif (ptr->getStartBuffPtr() != (uint8_t *)m_%s) {" % (self.getIndentation(1), param_name) )
```

Comment:
Why did you remove it? Moran's change fixes the bug in this code

Reply:
I removed it since it is not correct, parsing fails for list of classes of multiple entries. Once Moran's change is merged, I'll rebase and we'll get it back.

Reply:
Brought it back